Examples/AblyPush/AblyPushExample/AblyPushExample.entitlements

```diff
@@ -4,5 +4,11 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.location.push</key>
+	<true/>
+	<key>com.apple.security.application-groups</key>
```

Comment:
It doesn't seem necessary for a user who just wants to test normal push notifications to need to set up a security group. See other comment about perhaps having separate targets.

Reply:
(But need to make sure that in the case where security groups are not configured, the location push events view behaves gracefully — i.e. doesn't cause the app to crash.)

Reply:
https://github.com/ably/ably-cocoa/commit/14cd8d59cbe9a9fe539435923f82933e30ea14c7